app/actions/ModActionPanel/QuickAction.tsx

```diff
@@ -499,7 +501,12 @@ function Form(
 
                   {subjectStatus.lastReviewedAt ? (
                     <span className="pl-1">
-                      Last reviewed at:{' '}
+                      Last{' '}
+                      {subjectStatus.reviewState ===
+                      ToolsOzoneModerationDefs.REVIEWNONE
+                        ? 'event'
+                        : 'reviewed'}{' '}
+                      at:{' '}
                       {dateFormatter.format(
                         new Date(subjectStatus.lastReviewedAt),
```

Comment:
Is `lastReviewedAt` set when the review state is `REVIEWNONE`?

Reply:
yep.